### DISCUSSION TITLE
Documentation Guidance

### DISCUSSION THREAD

**Discussion by dev-marioed** [twbs/bootstrap#39153] [Q&A]
Hi. I've tried to read the official bootstrap documentation and i feel lost because either it has several links that take me to other applications like popper or just says "copy this on your htlm and you're set" How should I get involved into bootstrap? Should I take a Udemy course first? thanks

**Comment by ascensus-mdb** [comment]
Try this tutorial for example. It will guide you step by step  ;) 

https://mdbootstrap.com/learn/mdb-foundations/basics/introduction/
